### PR TITLE
Replace argument by parameter in image.py

### PR DESCRIPTION
**Pull request by michaelgrund**
Replaced  all appearances of _argument_ by _parameter_.

### CODE REVIEW

File: examples/gallery/plot/image.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 The :meth:`pygmt.Figure.image` method can be used to read and
 place a raster image file or an Encapsulated PostScript file
 on a map. We must specify the file as *str* via the ``imagefile``
-argument or simply use the filename as the first argument. You can
+parameter or simply use the filename as the first parameter. You can
```
Comment (weiji14) [inline examples/gallery/plot/image.py:7]:
```suggestion
parameter, or simply use the filename as the first argument. You can
```
I feel like this sentence is repeating itself somehow. The str input is an argument, which is passed to the `imagefile` parameter. Alternatively, the filename (also a str) can be passed in as an argument to the function?!

Comment (michaelgrund) [reply examples/gallery/plot/image.py]: You're right, thanks!

**Comment by seisman** [comment]
Just a quick comment: I don't quite like this example, because it's so similar to the "logo" example (https://www.pygmt.org/dev/gallery/plot/logo.html). Perhaps we can change to a better image?

**Comment by michaelgrund** [comment]
> 
> 
> Just a quick comment: I don't quite like this example, because it's so similar to the "logo" example (https://www.pygmt.org/dev/gallery/plot/logo.html). Perhaps we can change to a better image?

Agree with you @seisman , any suggestions which image we could use instead? Originally I selected this image since it is availabe from the GMT website, so we would know if its origin changes. Alternatively, we could store an example image in the repo.

**Comment by seisman** [comment]
Close this one as the changes has been merged into #943. We may still want to improve the example using a better figure.

**Pull request closed without merging**